### ISSUE TITLE
Extension loaded in BurpSuite but not working

### ISSUE THREAD

**Issue by cybernova** [sting8k/BurpSuite_403Bypasser#6]
Hi,

I've loaded the extension in BurpSuite with no errors, but when requesting a resource with a 403 response, I don't see any other requests in the Proxy HTTP History.

I'm doing something wrong?

Thanks

**Comment by sting8k** [comment]
You ‘re doing fine. Normally, requests from any extension will not be logged to HTTP History. You can check Extender -> Output of any extension for more info.

**Comment by cybernova** [comment]
I don't see any output in the Extender -> Output section of the extension either.

**Comment by sting8k** [comment]
> I don't see any output in the Extender -> Output section of the extension either.

Did you choose the extension you want to check output? Then you can refresh 403 page, and recheck output tab.

**Comment by cybernova** [comment]
Yes, I've done many times what you describe but still not working. No output and no error.

![burp1](https://user-images.githubusercontent.com/5947674/101237702-2fa4cf80-36db-11eb-9db8-1abfc1b8128f.png)


I'm using BurpSuite v2020.9.1 on Kali Linux

**Comment by MMquant** [comment]
@cybernova could you install Flow plugin and check the Burp outgoing traffic? (IMPORTANT: Flow plugin must be last item in "Burp extensions" list)

**Comment by cybernova** [comment]
@MMquant Ok, I tried but I don't see any useful output in Flow plugin tab. I've refreshed the 403 page, intercepted by the proxy but nothing.

![burp2](https://user-images.githubusercontent.com/5947674/101237875-6cbd9180-36dc-11eb-8622-e9e959c541a8.png)

**Comment by MMquant** [comment]
@cybernova could you

- Screenshot your dashboard tab
- Visit 403 page in browser
- Screeshot the Flow tab (not Flow in Extender ... Flow has its own tab)

**Comment by cybernova** [comment]
@MMquant Sorry I didn't see the Flow tab, I was focusing on in the Extender.

1)
![burp3](https://user-images.githubusercontent.com/5947674/101238147-84961500-36de-11eb-9a36-1ffc8803ce7e.png)

2)
![burp4](https://user-images.githubusercontent.com/5947674/101238153-8cee5000-36de-11eb-9be1-7a5c751a9fcf.png)

You can see the 403 resource requested.

**Comment by sting8k** [comment]
@cybernova i see, try to add “Live audit from proxy”. 
Guide:
[https://portswigger.net/burp/documentation/desktop/scanning/live-scans](https://portswigger.net/burp/documentation/desktop/scanning/live-scans)

**Comment by cybernova** [comment]
@sting8k Live audit can be enabled only with the Pro version which I don't have.

**Comment by sting8k** [comment]
@cybernova Try this: Right-click on any 403 request, Send to Passive scan. I am not sure if it is available on community version.

**Comment by cybernova** [comment]
@sting8k Nope, all the scan activities are not available for the Community version. So you guys tested the extension on the Pro version?

**Comment by sting8k** [comment]
@cybernova Yeah, I am using Pro version. I think you can use another tool like: [https://github.com/lobuhi/byp4xx](https://github.com/lobuhi/byp4xx)

**Comment by cybernova** [comment]
@sting8k Yes, I use that tool now. Thank you anyway

**Comment by zinminphyo0** [comment]
i can't install it, how to fix?
![2020-12-06_02-34](https://user-images.githubusercontent.com/74085414/101274457-3e879280-376c-11eb-8372-d011463dfd48.png)
![2020-12-06_02-38](https://user-images.githubusercontent.com/74085414/101274459-40e9ec80-376c-11eb-940a-bf61b648eed6.png)

**Comment by cybernova** [comment]
@zinminphyo0 you have to use not the Jython-installarer.jar but the Jython-standalone.jar version

**Comment by zinminphyo0** [comment]
It working , thank you so much @ cybernova
